explorer/nodewatch/openapi/openapi.yaml

```diff
@@ -0,0 +1,436 @@
+openapi: 3.0.3
+info:
+  title: NodeWatch API Documentation
+  description: >
+    API providing information about NEM and Symbol blockchain networks, including nodes, network heights, and time-series node statistics.
+  version: 1.0.0
+
+servers:
+  - url: https://nodewatch.symbol.tools/testnet
+    description: Testnet server
+  - url: https://nodewatch.symbol.tools
+    description: Mainnet server
+
+tags:
+  - name: NEM nodes
+    description: Endpoints providing NEM node and height information.
+  - name: Symbol nodes
+    description: Endpoints providing Symbol node, height, and statistics.
+
+paths:
+  /api/nem/nodes:
+    get:
+      summary: Returns the list of known NEM nodes
+      tags:
+        - NEM nodes
+      operationId: getNemNodes
+      responses:
+        200:
+          description: The list of known NEM nodes
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Node'
+
+  /api/nem/height:
+    get:
+      summary: Returns NEM heights
+      tags:
+        - NEM nodes
+      operationId: getNemHeight
+      responses:
+        200:
+          description: Height and finalized height
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HeightInfo'
+
+  /api/nem/nodes/count:
+    get:
+      summary: Returns NEM node count time series
+      tags:
+        - NEM nodes
+      operationId: getNemNodesCount
+      responses:
+        200:
+          description: Time series of node counts
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TimeSeriesNodesCount'
```

Comment:
in the NEM time-series node count, roles are not included.
The output looks like this:

```
{
    "date": "2025-08-13",
    "values": {
      "total": 5
    }
  },
  ```

Reply:
Yes. There is information in the `required` section that only the `total` is required. I added now also information to the description.